tracing-appender/src/rolling/builder.rs

```diff
@@ -208,6 +208,8 @@ impl Builder {
     /// Files matching these criteria may be deleted if the maximum number of
     /// log files in the directory has been reached.
     ///
+    /// Panics if and only if `n` is 0.
```

Comment:
style nit: elsewhere in the `tracing` API documentation, we try to use a `Panics` heading when documenting panic conditions:
```suggestion
    /// # Panics
    ///
    /// This method panics if and only if `n` is 0.
```